provisioner/kuberpak/internal/convert/registryv1.go

```diff
@@ -109,12 +109,12 @@ func Convert(in RegistryV1, installNamespace string, targetNamespaces []string)
 			},
 		}
 	}
-	var (
-		roles               []rbacv1.Role
-		roleBindings        []rbacv1.RoleBinding
-		clusterRoles        []rbacv1.ClusterRole
-		clusterRoleBindings []rbacv1.ClusterRoleBinding
-	)
+
+	roles := []rbacv1.Role{}
+	roleBindings := []rbacv1.RoleBinding{}
+	clusterRoles := []rbacv1.ClusterRole{}
+	clusterRoleBindings := []rbacv1.ClusterRoleBinding{}
```

Comment:
This was outlined by golangci-lint? The changes here should be identical to the original variable declarations.

Reply:
Yeah it was outlined by `prealloc` linter. I was debating just removing that linter for this reason since this felt kinda strange.

Reply:
Yeah this feels strange - I think I would be favor of dropping it. WDYT?

Reply:
To be fair, this would also satisfy the linter:

```go
var (
	roles        []rbacv1.Role
	roleBindings []rbacv1.RoleBinding
)

clusterRoles := []rbacv1.ClusterRole{}
clusterRoleBindings := []rbacv1.ClusterRoleBinding{}
```

Reply:
I'm also good with dropping it, sorry Github comments didn't update for a bit so I didn't see your message.